2-ui/5-loading/02-script-async-defer/article.md

```diff
@@ -192,7 +192,7 @@ Please note that if you're using `defer`, then the page is visible *before* the
 
 So the user may read the page, but some graphical components are probably not ready yet.
 
-There should be "loading" indication in proper places, not-working buttons disabled, to clearly show the user what's ready and what's not.
+There should be a "loading" indication in proper places, set not-working buttons to disabled, to clearly show the user what's ready and what's not.
```

Comment:
If it's "places" then it should be ' ..should be "loading" indication**s** '. (And if it's singular ("place"), then ' a "loading" indication in **a** (or **the**) proper place '